Sources/AWSLambdaRuntime/Lambda+Codable.swift

```diff
@@ -130,3 +131,17 @@ extension JSONEncoder: LambdaCodableEncoder {
         return buffer
     }
 }
+
+extension JSONEncoder {
+    /// Convenience method to allow encoding json directly into a String. It can be used to encode a payload into an APIGateway.V2.Response's body.
+    public func encodeAsString<T: Encodable>(_ value: T) throws -> String {
```

Comment:
can be refactored as
public func encodeAsString<Element: Encodable>(_ value: Element) throws -> String